pkgs/tools/networking/yggdrasil/default.nix

```diff
@@ -0,0 +1,36 @@
+{ pkgs, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "yggdrasil";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "yggdrasil-network";
+    repo = "yggdrasil-go";
+    rev = "v${version}";
+    sha256 = "0cbj9hqrgn93jlybf3mfpffb68yyizxlvrsh1s5q21jv2lhhjcwj";
+  };
+
+  modSha256 = "1r942mqlq9lqxizxi33768jp4j706mwa2fliz0qbkh98r4ri2292";
+
+  subPackages = [ "." ];
```

Comment:
```suggestion
```

Reply:
this yields this error:
`contrib/config/yggdrasilconf.go:65:14: cannot use flag.Arg(1) (type string) as type []string in assignment`

Reply:
I copied and pasted that line from the buildGoModule example in the Nixpkgs manual, without any idea what it does (and the manual is not helpful on that point).  When I remove it the package builds without error.  NixOS 19.03 x86_64-linux, @Lassulus  what are you building this on?

Reply:
I got this when I also removed the custom buildPhase. It works though if I add only
```
  subPackages = [ "cmd/yggdrasil" ];
```
or
```
  subPackages = [ "cmd/yggdrasilctl" ];
```
but not when both or none are present

Reply:
Thanks for figuring out that it worked with only one thing in subPackages.  That allowed me to track the problem down to a bug in buildGoModule, which I just submitted as #63923.